### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [gjoseph92/stackstac#8] [General]
I tried it out, seemed to work nicely.

The single machine case blew up when I expected it to on the 'compute stuff' side but still need to tinker more.

e.g. make some script examples to throw at an ec2 machine

**Comment by RichardScottOZ** [comment]
Streamlined things happily it seems, very nice work.

**Reply by RichardScottOZ** [reply]
It looks like something I was testing earlier, was taking 15-18 minutes a band to compute, this is doing 5-6, which seems rather impressive.

**Comment by RichardScottOZ** [comment]
For a single bad compute 40000*20000 approx, processed 308 times
in slightly longer than 36 times with an older method

so that is pretty great

**Reply by gjoseph92** [reply]
Awesome! So sounds like you're basically seeing a 10x speedup over what you had before?

Is this all running locally right now, or on AWS? I've noticed a huge difference when you can move the compute closer to the data. Particularly since GDAL usually needs to read a bit more data from the files than you end up using (depending on how the blocks overlap).

If you want a quick way to try it out on AWS, I do recommend https://cloud.coiled.io 😉

**Comment by RichardScottOZ** [comment]
r524x.large on AWS

and yeah, their bandwidth in US West 2 near the data - massive difference (apart from being much faster in general than garden variety NBN here).

**Comment by RichardScottOZ** [comment]
Yes, if I want someone to Dask cluster something for me, then quite possibly!
